source/includes/steps-starting-vsce-connect-with-advanced-connection-settings.yaml

```diff
@@ -246,6 +267,50 @@ content: |
                     For example, the username "X509User" must be 
                     provided as "CN=X509User".
 
+             - id: ldap
+               name: LDAP
+               content: |
+
+                 Select :guilabel:`LDAP` if the deployment
+                 uses :manual:`LDAP </core/security-ldap-external/>`
+                 as its authentication mechanism. If selected, you
+                 must provide the :guilabel:`Username` and
+                 :guilabel:`Password` to authenticate
+                 the user.
+
+             - id: AWS-IAM
+               name: AWS IAM
+               content: |
+
+                 Select AWS IAM if the deployment uses 
+                 `AWS IAM <https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html>`__ 
+                 as its authentication mechanism.
+
+                 .. list-table::
+                    :header-rows: 1
+                    :widths: 40 20 40
+
+                    * - Field
+                      - Necessity
+                      - Description
+
+                    * - AWS Access Key Id
+                      - Required
+                      - An access key ID is a unique identifier 
+                        associated with your AWS account.
+
+                    * - AWS Secret Access Key
+                      - Optional
+                      - The secret access key is a long-term 
+                        credential associated with the access key ID.
+
+                    * - AWS Access Key Id
```

Comment:
We have AWS Access Key Id twice. Is that correct?

Reply:
Yes, this is suppose to be session token, good catch.